spid-batch-import/spid-idp-spimport.js

```diff
@@ -27,7 +27,7 @@ importSp(data, (result) => {
     process.stdout.write("Service Provider successfully imported\n");
   } else {
     process.stdout.write("Error\n");
-    process.stdout.write(result + "\n");
+    process.stdout.write(JSON.stringify(result) + "\n");
```

Comment:
is there any reason on why we don't use console.log (which supports printf format strings and json dumps ?)

Reply:
Code is copied from https://github.com/italia/spid-testenv-backoffice/blob/master/user-import/spid-idp-userimport.js which is very ugly but at the time it was the fastest way to setup the local IDP :-/